style.css

```diff
@@ -405,11 +431,151 @@ p {
   border: 0;
 }
 
-#btn-f:hover {
+#form button:hover {
   background-color: #7f8cff;
   color: #ebebff;
 }
 
-#btn-f:active {
+#form button:active {
   background-color: #2230d2;
 }
+
+@media (min-width: 768px) {
+  .burger-icon {
+    display: none;
+  }
+
+  .ul {
+    display: flex;
+  }
+
+  .main {
+    background-image: url("img/Header\ bg@2x.png") !important;
+    background-size: contain;
+  }
+
+  #work-section {
+    grid-row-gap: 2.2em;
+  }
+
+  .cards {
+    display: flex;
+    flex-direction: row;
+    gap: 2%;
+    margin-top: 8em;
+    min-width: 82vw;
+    min-height: 54vh;
+  }
+
+  #cards2 {
+    flex-direction: row-reverse;
+  }
+
+  #cards4 {
+    flex-direction: row-reverse;
+  }
```

Comment:
#6 Here I kindly recommend using a class in the HTML tags that you need to use the row-reverse, so you can reduce the redundancy.